docs/modules/reference/pages/configuration/tuning-activemq.adoc

```diff
@@ -50,7 +50,35 @@ In order to isolate multiple instances on the same broker, you can customize the
 echo 'org.opennms.instance.id=MyNMS' > "$\{OPENNMS_HOME}/etc/opennms.properties.d/instance-id.properties"
 ----
 
-CAUTION: If you change the instance id setting when using the embedded broker, you will need to update the authorization section in the broker's configuration to reflect the updated prefix.
+Update minion's instance id accordingly to match the OpenNMS instance's. The same id instance property should be present in ${MINION_HOME}/etc/custom.system.properties.
```

Comment:
Swap to use the product name (Meridian/Horizon) and split sentences on to separate lines.
```suggestion
Update minion's instance ID accordingly to match the {page-component-title} instance.
The same ID instance property should be present in `${MINION_HOME}/etc/custom.system.properties`.
```